iprepd.go

```diff
@@ -41,6 +42,9 @@ type serverCfg struct {
 		AWS  bool
 	}
 	VersionResponse string
+	Statsd          struct {
```

Comment:
Looks like this gofmt here is off?

Reply:
No, but it does look weird. It's just being aligned with the `VersionResponse` above it.